Magento_Reward/templates/customer/reward/history.phtml

```diff
@@ -56,7 +56,7 @@
 
                             <td data-th="<?= /* @escapeNotVerified */ $block->escapeHtml(__('Reason')); ?>" classs="col reason">
                                 <?= /* @escapeNotVerified */ $block->escapeHtml($block->getMessage($item)) ?>
-                                
+
```

Comment:
I cannot post it on the proper line , so I have to comment here: There are multiple typos in `class` attribute of `td` elements on lines 45-67

Reply:
thanks! nice catch!